zulipterminal/config/themes.py

```diff
@@ -255,6 +261,28 @@ def parse_themefile(
     return urwid_theme
 
 
+def translate_styles(
+    styles: Dict[Any, str], translations: Dict[str, str] = STYLE_TRANSLATIONS
+) -> Dict[Any, str]:
```

Comment:
I have improved the `Any` here. I can refactor for other occurrences of `Any` in themes. Should I proceed with that too? (I am not sure if it will be relevant to this issue)

Reply:
Let's leave other refactors for another PR - there are various possibilities, but getting this fixed would be useful.